includes/RestAdmin.php

```diff
@@ -173,26 +173,26 @@ public function settings()
     {
 
         register_setting(
```

Comment:
@gassius I think, now that we've switched to constants, we don't need those options anymore, right?

Reply:
commit 5bd106d should resolve it